assignments/objects.js

```diff
@@ -49,16 +103,49 @@ const example = {
 // 3. Nest a grandchild object in the child object with properties for name and age.  The name will be Sam and the age will be 30
 // 4. Give each of the objects the ability to speak their names using the this keyword.
 
-const parent = {}
+const parent = {
+  "name":"Susan",
+  "age":70,
+  "speak": function() {
+    return `My name is ${this.name}.`;
+  },
+  "child":{
+    "name":"George",
+    "age":50,
+    "speak": function() {
+      return `My name is ${this.name}.`;
+    },
+    "grandchild":{
+      "name":"Sam",
+      "age":30,
+      "speak": function() {
+        return `My name is ${this.name}.`;
+      }
+    }
+  }
+}
```

Comment:
Good job here. This will even be useful when working with databases so don't forget this.